lib/OpenQA/Schema/Result/Jobs.pm

```diff
@@ -1525,7 +1525,20 @@ sub _find_network {
 sub release_networks {
     my ($self) = @_;
 
-    $self->networks->delete;
+    my @cluster = keys %{$self->cluster_jobs};
+    return $self->networks->delete unless @cluster;
```

Comment:
Because that was done on top of this set of changes https://github.com/os-autoinst/openQA/pull/1729/commits/d73f4f44a7170fdfc152c4602ae0263e70feb673 which was reverted, but it's needed  to release correctly networks only when the last job in the cluster finishes.. otherwise another MM test can take the same vlan (as we delete the network, thus it's available for new jobs) while others could be still using it

Reply:
I'm afraid I still don't understand: is it ok in this branch or shall I do something about it?

Reply:
It's fine to keep it here

Reply:
thanks for clarifying

Reply:
ok, this is really problematic - because the return value of cluster_jobs changes if you cloned parts.

Reply:
See ac9f540ef945520ae209e0298f3a37487ec71eb9 for a more reliable fix